FindNearby/force-app/main/default/lwc/mobileMapLayersMain/overrideCSS.js

```diff
@@ -6,6 +6,59 @@ body, div, button, input, optgroup, select, textarea, p, ul, li {
     font-family: var(--lwc-fontFamily) !important;
 }
 
+.confirm-modal-container lightning-modal {
```

Comment:
This is a CSS file but is saved as a JS file `overrideCSS.js` ?!!

Reply:
This is a file to control CSS of lightning components (`lightning-modal`, `lightning-datepicker` etc.)
The LWC team told me this is the only way to control inner parts of the components, otherwise there is no access.
I add this file in a <style> tag to the <head> tag of the HTML.

Reply:
Oh I see... I learned a new thing today 🙂